cpp/src/gandiva/precompiled/timestamp_arithmetic.cc

```diff
@@ -239,4 +290,18 @@ ADD_TIMESTAMP_TO_INT64_FIXED_UNITS(date64, add, MILLIS_IN_DAY)
 ADD_TIMESTAMP_TO_INT64_FIXED_UNITS(timestamp, date_add, MILLIS_IN_DAY)
 ADD_TIMESTAMP_TO_INT64_FIXED_UNITS(timestamp, add, MILLIS_IN_DAY)
 
+// add timestamp to day time interval
```

Comment:
Is this comment correct? The function is adding a day time interval in timestamp, right?